### PR TITLE
Isogram approach

### PR DESCRIPTION
**Pull request by bobahop**
Approach for Isogram.

### CODE REVIEW

**Comment by bobahop** [comment]
@wolf99 and/or @ErikSchierboom: C Isogram approach ready for review.

File: exercises/practice/isogram/.approaches/introduction.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Introduction
+
+There are various idiomatic ways to solve Isogram.
+One approach can use a bit field to keep track of used letters.
+
+# General guidance
```
Comment (wolf99) [inline exercises/practice/isogram/.approaches/introduction.md:6]:
```suggestion
## General guidance
```

**Review by wolf99 — CHANGES_REQUESTED** [review]

File: exercises/practice/isogram/.approaches/introduction.md (hunk 1 above)
Comment (bobahop) [reply exercises/practice/isogram/.approaches/introduction.md:6]: Ugh! I don't know why I keep doing that! I know better!

**Comment by bobahop** [comment]
@wolf99: Thanks for the catch! Change is committed.

**Review by wolf99 — APPROVED** [review]

**Pull request merged**